### ISSUE TITLE
Create an index of commonly filtered values and show them on auto-complete inputs

### ISSUE THREAD

**Issue by edwinmp** [devinit/ddw-analyst-ui#530]
![image](https://user-images.githubusercontent.com/5672438/119539751-40cebf00-bd95-11eb-9c7f-efe9b9a92f4b.png)

We may need to initially pull them from existing datasets

**Comment by edwinmp** [comment]
@dean-breed @Duncan-Knox  this would be how we meet you halfway on your request.
@akmiller01 I think this is doable but I could use a second opinion.

**Comment by akmiller01** [comment]
Yes if the strings are indexed, it shouldn't be too heavy of a lift to `select distinct(column_x) from repo.dataset_y;`

If we wanted to try pushing the boundaries, we could even facet certain fields at any given step in the query. So if the query so far is `sq0` you could `select distinct(sq0.column_x) from (select xyz as sq0);`

But in either case, I think I would prefer a button on the UI that generates facet suggestions on click rather than running the facets every time a user starts typing.

**Comment by k8hughes** [comment]
@ThatcherK if you are working on this can you mark as in progress and put an update on the ticket?

**Comment by ThatcherK** [comment]
@k8hughes , no am not working on this, I forgot to update the ticket after dropping it, I was told it was not a priority
